### PR TITLE
:explain command using shared state for modal

### PR DESCRIPTION
**Pull request by chriscoyier**
For https://github.com/elanthia-online/illthorn/issues/9

### CODE REVIEW

File: src/app/ui/cli.js [hunk 1]
```diff
@@ -124,10 +121,19 @@ module.exports = class CLI {
...
-    CLI.HELP_UI_OPEN = !CLI.HELP_UI_OPEN
+    Session.focused().state._modals.commands = !Session.focused()
+      .state._modals.commands
   }
 
   view({ attrs }) {
+    let commandsModalOpen = false
+    if (typeof Session.focused() !== "undefined") {
+      if (
+        Session.focused().state._modals.commands === true
```
Comment (ondreian) [inline src/app/ui/cli.js:132]: This should use `Lens.get` and remove all the `if` statements and use a `const`

Comment (chriscoyier) [reply src/app/ui/cli.js:132]: I was trying to use optional chaining but that didn't seem to be working. `Lens` is clearly the way here, I'll look into getting that in as soon as I can.

Comment (chriscoyier) [reply src/app/ui/cli.js]: ok another crack at that is in

**Comment by ondreian** [comment]
Just fixing up some merge conflicts that happened while I went down the Electron upgrade rabbit hole for you.  Thanks for another contrib!

**Pull request merged**